### ISSUE TITLE
sysctl module always shows false positive results

### ISSUE THREAD

**Issue by maratsh** [hubblestack/hubble-salt#77]
With default top and hubblstack 2017.4.1, this is reproducible for all sysctl modules
```
nova:
  'G@osfinger:*CoreOS*':
    - cis.coreos-level-1
  'G@osfinger:CentOS-6':
    - cis.centos-6-level-1-scored-v2-0-1
  'G@osfinger:CentOS*Linux-7':
    - cis.centos-7-level-1-scored-v2-1-0
  'G@osfinger:Debian-8':
    - cis.debian-8-level-1-scored-v1-0-0
  'G@osfinger:Red*Hat*Enterprise*Linux*Server-6':
    - cis.rhels-6-level-1-scored-v2-0-1
  'G@osfinger:Red*Hat*Enterprise*Linux*Server-7':
    - cis.rhels-7-level-1-scored-v2-1-0
  'G@osfinger:Red*Hat*Enterprise*Linux*Workstation-7':
    - cis.rhelw-7-level-1-scored-v2-1-0
  'G@osfinger:Ubuntu-14.04':
    - cis.ubuntu-1404-level-1-scored-v1-0-0
  'G@osfinger:Ubuntu-16.04':
    - cis.ubuntu-1604-level-1-scored-v1-0-0
  'G@osfullname:Microsoft*Windows*Server*2008*':
    - cis.windows-2008r2-level-1-scored-v3-0-0
  'G@osfullname:Microsoft*Windows*Server*2012*':
    - cis.windows-2012r2-level-1-scored-v2-0-0
  'G@osfinger:Amazon*Linux*2014*':
    - cis.amazon-201409-level-1-scored-v1-0-0
  'G@osfinger:Amazon*Linux*2015*':
    - cis.amazon-level-1-scored-v1-0-0
  'G@osfinger:Amazon*Linux*2016*':
    - cis.amazon-level-1-scored-v1-0-0


```
```
salt-call  hubble.top verbose=true show_success=False

          CIS-7.2.7:
              ----------
              description:
                  Enable RFC-recommended Source Route Validation
              match_output:
                  1
              module:
                  sysctl
              name:
                  net.ipv4.conf.default.rp_filter
              nova_profile:
                  debian-8-level-1-scored-v1-0-0
              tag:
                  CIS-7.2.7
```
The system have net.ipv4.conf.default.rp_filter =1 
```
salt-call sysctl.get net.ipv4.conf.default.rp_filter
[INFO    ] Executing command 'sysctl -n net.ipv4.conf.default.rp_filter' in directory '/root'
local:
    1
```
```
salt-call --versions-report
Salt Version:
           Salt: 2016.11.5
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.2
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.3
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.2
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.9 (default, Mar  1 2015, 12:57:24)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.4.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
 
System Versions:
           dist: debian 8.4 
        machine: x86_64
        release: 3.16.0-4-amd64
         system: Linux
        version: debian 8.4 


```

**Comment by basepi** [comment]
Thanks for the report, we'll definitely look into this.

**Comment by mew1033** [comment]
Is this the same as #84? If so, it should be fixed by #85.

**Comment by basepi** [comment]
Looks likely. @maratsh is `net.ipv4.conf.all.rp_filter` also set to 1? That check actually checks for both, and if either fails it was reporting the last one even if it wasn't the one to fail. It's been fixed in #85.

**Comment by basepi** [comment]
I'm going to assume this is fixed by #85, but please comment if it's not and we can re-open!
